tests/package.json

```diff
@@ -0,0 +1,185 @@
+{
+ "name": "zulip",
+ "productName": "ZulipTest",
+ "version": "2.3.3",
+ "main": "../app/main",
+ "description": "Zulip Desktop App",
```

Comment:
For making the electron app use the `ZulipTest` app temp directory and the `../app/main` entry point only `productName`, `name` and `version` are required, you can remove other stuff.

Reply:
@priyankp10  I think the a good idea would be to remove this file, and create it at the time of test start and then remove it when the test is over. What do you think?